bookmarket-website/src/pages/LoginPage.tsx

```diff
@@ -1,12 +1,54 @@
-import { NavBar } from "@/components/NavBar";
 import "./LoginPage.css";
 
+import { NavBar } from "@/components/NavBar";
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+import { useState } from "react";
+
 export function LoginPage() {
+    const [inputtedEmail, setInputtedEmail] = useState("");
+    const [inputtedPassword, setInputtedPassword] = useState("");
+
     return (
-        <div>
+        <div className="login-page">
             <NavBar />
 
-            <h1>Login</h1>
+            <h1 className="page-title">Login</h1>
```

Comment:
Let's add some text here that says, "Don't have an account? Click here to sign up" where `Click here` is a link to the /signup page. It would look something like
```
<p>Don't have an account? <Link to="/signup">Click here</Link> to sign up</p>
```
We will need to use that special `Link` component provided by the `react-router`dom` package since we're using React here.